actionpack/lib/action_dispatch/system_testing/driver.rb

```diff
@@ -23,15 +23,19 @@ def use
       end
 
       private
+        def name
+          @name ||= "#{@driver_type}-#{object_id}"
+        end
+
```

Comment:
One more thought - in https://github.com/rails/rails/issues/32468, @Edouard-chin had proposed that `driven_by` could be given an additional parameter that allows consumers specify the name to use when registering the driver with Capybara. Is that something that we should consider? (It could be an optional parameter, falling back to the `#{driver}_#{using}_#{object_id}` default.)

It might make the "constructing a `Capybara::Session` manually" use case easier since the consumer could simply pass the symbol they used in the original `driven_by` declaration, rather than getting the `ActionDispatch::SystemTesting::Driver` instance and calling `name` on it.

Reply:
Implemented the ability the specify the driver name explicitly in 97313e0.